packages/wallets/src/components/offers/OfferSummary.tsx

```diff
@@ -107,11 +110,35 @@ export default function OfferSummary(props: Props) {
           <OfferExchangeRate makerAssetInfo={makerAssetInfo} takerAssetInfo={takerAssetInfo} makerExchangeRate={makerExchangeRate} takerExchangeRate={takerExchangeRate} />
         </Flex>
       )}
+      {makerFee > 0 && (
+        <Flex flexDirection="column" gap={2}>
+          <Divider />
+          <Flex flexDirection="row" alignItems="center" gap={1}>
+            <Typography variant="body1" color="secondary" style={{fontWeight: 'bold'}}>
+              <Trans>Fees included in offer:</Trans>
+            </Typography>
+            <Typography color="primary">{Number(makerFee).toLocaleString()}</Typography>
```

Comment:
toLocaleString can be replaced with component <FormatLargeNumbers value={makerFee} />